test-app/tests/integration/components/lottie-test.ts

```diff
@@ -16,11 +13,12 @@ interface TestContext extends TestContextBase {
 
 module('Integration | Component | lottie', function (hooks) {
   setupRenderingTest(hooks);
-  setupWindowMock(hooks);
```

Comment:
All tests are passing without this. 
Moreover `fetch` is not present in the `window` mock in order to spy on.

Reply:
@mrloop sinon spying and stubbing will work now. 
There was an issue with latest sinon and ember-window-mock proxy implementation. It has been fixed in the latest version here: https://github.com/simonihmig/ember-window-mock/pull/535

Reply:
Great, thank you!